src/catalog/catalog-flat.md

```diff
@@ -13,71 +14,71 @@ Before you enable a flat catalog for a live store, make sure to test the configu
 
 ## Step 1: Enable the Flat Catalog
 
-1. On the Admin sidebar, tap **Stores**. Then under **Settings**, choose **Configuration**.
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
```

Comment:
Should you use "click" instead of "go to"? You can also use "navigate to"?

Reply:
Landed on "go to" for simplicity's sake... One of those things we need to standardize in some way. Our nav UI is kind of a combination hover/click.

Reply:
Yeah I get that. Not bolding the unclickable areas seems to be pretty apparent though, so "click" would make more sense. Def something to consider for the future.